csrc/ir/nodes.cpp

```diff
@@ -5155,12 +5155,15 @@ ForLoop::ForLoop(
   NVF_ERROR(isIntegralType(index->dtype()));
   addInput(index);
   addInput(iter_domain);
-  if (start == nullptr && iter_domain->isThread()) {
+  if (start == nullptr &&
+      (iter_domain->isThread() || iter_domain->isStream())) {
     start = NamedScalar::getParallelIndex(iter_domain->getParallelType());
   }
   if (step == nullptr) {
     if (iter_domain->isThread()) {
       step = NamedScalar::getParallelDim(iter_domain->getParallelType());
+    } else if (iter_domain->isStream()) {
+      step = iter_domain->extent();
```

Comment:
nitpicking: I think this could use a comment.

IIRC, we are not using the steps for stream parallelization at all. So this is just a stub for now?

Reply:
Done -- added a comment